### PR TITLE
fix(data): Fixed doc ordering and added attribute readme

### PR DESCRIPTION
**Pull request by SurgeCode**

### CODE REVIEW

File: packages/storage/README.md [hunk 1]
```diff
@@ -8,6 +8,76 @@ Currently we support Arweave via hosted service. **You will need a new API key f
 
 Please get in touch through Telegram if you are interested in using the new service.
 
-### Coming soon...
+# Arweave
```
Comment (cif) [inline packages/storage/README.md:11]: ❤️

**Review by cif — APPROVED** [review]

**Review by sainthiago — APPROVED** [review]

File: packages/react/README.md [hunk 2]
```diff
@@ -110,5 +98,18 @@ const  NearWalletConnector = () => {
   )
 }
 
+
+# Hooks
+
+**Coming soon!** a variety of hooks to make data fetching and transactions from our SDK/Data modules super easy in your React apps.
```
Comment (rubenmarcus) [inline packages/react/README.md:104]: cool!

File: packages/data/src/api/attributes/README.MD [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+{% endhint %}
+
+
+
+
+Returns attributes of a token by `metadataId`.
+
+
+
+### attributesBymetaId(metadataId: string)
```
Comment (rubenmarcus) [inline packages/data/src/api/attributes/README.MD:20]: cool!

**Review by rubenmarcus — APPROVED** [review]
cool! just need to resolve conflicts and its good to go!

**Pull request merged**